TypeCobol.LanguageServer/TypeCobolServer.cs

```diff
@@ -63,6 +66,12 @@ public TypeCobolServer(IRPCServer rpcServer, Queue<MessageActionWrapper> message
         /// </summary>
         public bool UseEuroInformationLegacyReplacingSyntax { get; set; }
 
+#if EUROINFO_RULES
+        /// <summary>
+        /// The Instance of the Cpy Copy names Map
+        /// </summary>
+        public CopyNameMapFile CpyCopyNamesMap { get; set; }
```

Comment:
No need to have the real object at this point, should be a `string`.

Reply:
done in e4234a0